src/mixins/nft.js

```diff
@@ -392,14 +392,24 @@ export default {
     },
     async collectNFT() {
       try {
-        logPurchaseFlowEvent(this, 'add_to_cart', {
+        const purchaseEventParams = {
           name: this.NFTName,
           price: this.purchaseInfo.price,
           classId: this.classId,
-        });
-        await this.initIfNecessary();
-        this.fetchUserCollectedCount();
-        this.walletFetchLIKEBalance();
+        };
+        logPurchaseFlowEvent(this, 'add_to_cart', purchaseEventParams);
+        logPurchaseFlowEvent(this, 'begin_checkout', purchaseEventParams);
+        logPurchaseFlowEvent(this, 'add_shipping_info', purchaseEventParams);
```

Comment:
I think user without wallet has no `shipp_info` since you ship nft to their wallet